.github/workflows/reusable-factory.yaml

```diff
@@ -234,6 +234,21 @@ jobs:
             fi
           fi
           
+          # Download Dockerfile.img only if RAW builds are requested
+          if [[ "${{ inputs.raw }}" == "true" ]]; then
+            echo "📥 Downloading Kairos Dockerfile.img for RAW builds from https://github.com/kairos-io/kairos/blob/${{ inputs.kairos_version }}/images/Dockerfile.img"
```

Comment:
What is `Dockerfile.img` ?

Reply:
@jimmykarily is just a simple Dockerfile to copy the artifact inside an oci image, it's the one that produces the quay.io/kairos/FLAVOR:FOOBAR-img